addons/account/models/account_move.py

```diff
@@ -1744,7 +1744,7 @@ def _check_unique_sequence_number(self):
                 AND move2.journal_id = move.journal_id
                 AND move2.move_type = move.move_type
                 AND move2.id != move.id
-            WHERE move.id IN %s AND move2.state = 'posted'
+            WHERE move.id IN %s AND move2.state = 'posted' AND move2.name <> '/'
```

Comment:
```suggestion
            WHERE move.id IN %s AND move2.state = 'posted' AND move2.name != '/'
```
I prefer `!=`  😁

Reply:
The official one is the one I have put:

https://www.postgresql.org/docs/9.0/functions-comparison.html

The other is supported as an alias, but converted on the fly to the one I have put. Anyway, if Odoo people express any preference, I can change it.